python/tank/bootstrap/manager.py

```diff
@@ -574,14 +593,109 @@ def _bootstrap_sgtk(self, engine_name, entity, progress_callback=None):
         else:
             raise TankBootstrapError("Unknown configuration update status!")
 
+        return config, status
+
+    def bootstrap_toolkit(self, entity):
```

Comment:
Unless there are good reasons not to, we should keep this internal. The bootstrapper should be as much of a black box as possible, only to expose the very minimum interface that we can. We used to have this method at one point but it was made internal. With bootstrap and tk plugins, the world is a lot simpler if there is always an engine running - bootstrapping into a free floating sgtk instance makes things more complicated. There were also very practical reasons why this didn't make sense (which i am trying to remember now 😛 ). Happy to discuss this in depth :)